lang/lang-pt_br.js

```diff
@@ -549,5 +590,6 @@
     TypeError: 'Sua Webcam não foi inicializada corretamente. Favor recarregar a página.',
     newPage: 'Inserir página',
   };
-  window.message = message;
+  // window.message = message;
+  window.congreaLanguages.pt_br = message;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment to property of function parameter 'window'.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)